core/api/src/domain/authorization/index.ts

```diff
@@ -0,0 +1,6 @@
+export const ScopesOauth2 = {
+  TransactionsRead: "transactions:read",
+  Offline: "offline",
```

Comment:
Where is Offline used?

Reply:
it's a Oauth2 convention things. it's basically whether the client should receive an refresh token in addition to the access token.

not sure that will be useful, but I added that as part of my R&D to hydra.